test_scripts/API/AddSubMenu/ATF_AddSubMenu_Negative_SubMenuIcon_OutUpperBound.lua

```diff
@@ -54,7 +54,7 @@ function Test:Precondition_ActivateApp()
   end)
   EXPECT_NOTIFICATION("OnHMIStatus", {hmiLevel = "FULL", systemContext = "MAIN", audioStreamingState = "AUDIBLE"})
 end
-commonSteps:PutFile("PutFile_strOutofBoundFileName", strOutofBoundFileName)
+-- commonSteps:PutFile("PutFile_strOutofBoundFileName", strOutofBoundFileName)
```

Comment:
@dboltovskyi, The test AddSubMenu_SubMenuIconOutUpperBound should verify that within default timeout(10 sec) OnHashChange will not come.
When using Times(0), ATF doesn't wait default timeout to expire.
Please add "commonTestCases:DelayedExp(10000)"